server/Makefile

```diff
@@ -32,11 +32,3 @@ e2e:
 	--start-directory tests/e2e \
 	--top-level-directory .. \
 	--verbose; test_result=$$?; \
-
-
-# Bundles the application so that it can be dockerized and run in ECS
-.PHONY: build-ecs
```

Comment:
isn't this used on line 32 of the dockerfiles

Reply:
No, I left `build-ecs` inside the `ecs` folder.